docs/init_with.md

```diff
@@ -0,0 +1,36 @@
+---
+id: init_with
+title: Initiate cloning target
+sidebar_label: Initialization block
+---
+
+You can override the default Clowne method which generates a _plain_ copy of a source object.
+By default, Clowne initiatas the clone record using a `#dup` method.
+
+For example, Cloners could be used not only to generate _fresh_ new models but to apply some transformations to the existing record:
+
+
+```ruby
+class User < ApplicationRecord
+  has_one :profile
+  has_many :posts
+end
+
+class UserCloner < Clowne::Cloner
+  adapter :active_record
+
+  include_association :profile
+
+  trait :copy_settings do
+    # Use a `target` for all the actions
+    init_with { |_source, target:| target }
```

Comment:
What do you think about `init_as` instead of `init_with`?

Reply:
Good idea